articles/urls.py

```diff
@@ -1,7 +1,10 @@
 from django.urls import path
 
-from articles.views import HomeView
+from articles.views import HomeView, PrivacyPoliceView
```

Comment:
```suggestion
from articles.views import HomeView, PrivacyPolicyView
```
Typo. Please change all places.